### ISSUE TITLE
Add a th element

### ISSUE THREAD

**Issue by shaunix** [projectmallard/projectmallard.org#9]
Mallard basically follows the HTML table model, deviating only for styling purposes (border, lines between, alternate shading). But it doesn't have a th element. th is good for accessibility, especially with row headers that can't be indicated with something like thead.

**Comment by shaunix** [comment]
http://projectmallard.org/mep/mep0012.html

**Comment by shaunix** [comment]
Spec written. http://projectmallard.org/1.1/mal_table_td.html
Already fully implemented in yelp-xsl.
